### ISSUE TITLE
hyperparameter setting?

### ISSUE THREAD

**Issue by byuns9334** [b-etienne/Seq2seq-PyTorch#2]
hi, when I train with hyperparameters which is exactly same with opened json setting (for bahdanau), i don't get as pretty attention plot as yours. DId you modify some hyperparameter from current setting?

**Comment by daofeng2007** [comment]
I got the same issue. The bahdanau got evaluation accuray 70.5% and never improved. The luong example leads to evaluation accuracy 0.0. Could @b-etienne point out the reason?
